test/test_cli.go

```diff
@@ -231,17 +242,18 @@ func (s *CLISuite) TestScale(t *c.C) {
 
 	// unchanged processes shouldn't appear in output
 	scale = app.flynn("scale", "echoer=3", "printer=0")
+	app.waitFor(ct.JobEvents{"printer": {"down": 2}})
 	t.Assert(scale, Succeeds)
 	t.Assert(scale, SuccessfulOutputContains, "printer: 2=>0")
-	t.Assert(scale, c.Not(SuccessfulOutputContains), "echoer")
+	t.Assert(scale, c.Not(OutputContains), "echoer")
```

Comment:
SuccessfulOutputContains is analogous to `Success AND OutputContains`. This means that `c.Not(SuccessfulOutputContains)` is analogous to `NOT Success OR NOT OutputContains`, when I want to explicitly check that the output doesn't contain a specific string.